include/json/features.h

```diff
@@ -52,6 +52,9 @@ class JSON_API Features {
 
   /// \c true if numeric object key are allowed. Default: \c false.
   bool allowNumericKeys_;
+
+  /// \c true if duplicate keys within an object should be rejected. Default: \c false.
+  bool rejectDupKeys_;
```

Comment:
Sorry. We cannot add fields to public classes. That could break binary compatibility.

This is, in fact, the main reason why we create `CharReader` and deprecated `Reader`. With `CharReader`, you can configure these settings via an actual JSON string:
* http://open-source-parsers.github.io/jsoncpp-docs/doxygen/class_json_1_1_char_reader_builder.html
* http://open-source-parsers.github.io/jsoncpp-docs/doxygen/index.html ("Advanced Usage")